swift/main.swift

```diff
@@ -0,0 +1 @@
+print("Hello, World!")
```

Comment:
Can you instead do a simple library with a trivial unit test, and run bazel test instead of bazel build?

Reply:
Let me try.